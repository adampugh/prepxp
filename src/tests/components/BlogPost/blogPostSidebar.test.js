import React from "react";
import { shallow } from "enzyme";
import BlogPostSidebar from "../../../components/BlogPost/blogPostSidebar";
import posts from "../../fixtures/blogPosts";

describe("<blogPostSidebar />", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<BlogPostSidebar posts={posts} />);
        expect(wrapper).toMatchSnapshot();
    });
});