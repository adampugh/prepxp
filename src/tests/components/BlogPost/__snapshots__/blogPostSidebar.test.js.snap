// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<blogPostSidebar /> should match snapshot 1`] = `
<div
  className="blogPost__sidebar__container"
>
  <div
    className="blogPost__sidebar"
  >
    <h1
      className="blogPost__sidebar__title"
    >
      Latest articles
    </h1>
    <Link
      key="1234"
      replace={false}
      to={
        Object {
          "pathname": "/prepxp/blog/5-Tips-to-get-things-done",
          "state": Object {
            "post": Object {
              "byline": "yesyesyeys",
              "id": "1234",
              "img": "http://via.placeholder.com/300x300",
              "text": "sf sdfg dfasd df dg asdf sdf adsf sadf yea oia the ruler is bakc ofi course is i sfsf aoosd sgodf sfoeg wefodfguonfls sdfsd fsd sgosfsd sdfpsdf sdgxfogfdng  sdfosf sgosf sdfosg sdosf sgosdf sdfosdg sfosdf sd",
              "title": "5 Tips to get things done",
            },
            "posts": Array [
              Object {
                "byline": "yesyesyeys",
                "id": "1234",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf yea oia the ruler is bakc ofi course is i sfsf aoosd sgodf sfoeg wefodfguonfls sdfsd fsd sgosfsd sdfpsdf sdgxfogfdng  sdfosf sgosf sdfosg sdosf sgosdf sdfosdg sfosdf sd",
                "title": "5 Tips to get things done",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "134t",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Pass the interview with these answers",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "fddh",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Top 10 most common questions",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "ergerg",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "What Google and employers are looking for",
              },
            ],
          },
        }
      }
    >
      <div
        className="blogPost__sidebar__item"
      >
        <h2>
          5 Tips to get things done
        </h2>
        <p />
        <hr />
      </div>
    </Link>
    <Link
      key="134t"
      replace={false}
      to={
        Object {
          "pathname": "/prepxp/blog/Pass-the-interview-with-these-answers",
          "state": Object {
            "post": Object {
              "byline": "yesyesyeys",
              "id": "134t",
              "img": "http://via.placeholder.com/300x300",
              "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
              "title": "Pass the interview with these answers",
            },
            "posts": Array [
              Object {
                "byline": "yesyesyeys",
                "id": "1234",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf yea oia the ruler is bakc ofi course is i sfsf aoosd sgodf sfoeg wefodfguonfls sdfsd fsd sgosfsd sdfpsdf sdgxfogfdng  sdfosf sgosf sdfosg sdosf sgosdf sdfosdg sfosdf sd",
                "title": "5 Tips to get things done",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "134t",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Pass the interview with these answers",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "fddh",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Top 10 most common questions",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "ergerg",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "What Google and employers are looking for",
              },
            ],
          },
        }
      }
    >
      <div
        className="blogPost__sidebar__item"
      >
        <h2>
          Pass the interview with these answers
        </h2>
        <p />
        <hr />
      </div>
    </Link>
    <Link
      key="fddh"
      replace={false}
      to={
        Object {
          "pathname": "/prepxp/blog/Top-10-most-common-questions",
          "state": Object {
            "post": Object {
              "byline": "yesyesyeys",
              "id": "fddh",
              "img": "http://via.placeholder.com/300x300",
              "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
              "title": "Top 10 most common questions",
            },
            "posts": Array [
              Object {
                "byline": "yesyesyeys",
                "id": "1234",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf yea oia the ruler is bakc ofi course is i sfsf aoosd sgodf sfoeg wefodfguonfls sdfsd fsd sgosfsd sdfpsdf sdgxfogfdng  sdfosf sgosf sdfosg sdosf sgosdf sdfosdg sfosdf sd",
                "title": "5 Tips to get things done",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "134t",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Pass the interview with these answers",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "fddh",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Top 10 most common questions",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "ergerg",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "What Google and employers are looking for",
              },
            ],
          },
        }
      }
    >
      <div
        className="blogPost__sidebar__item"
      >
        <h2>
          Top 10 most common questions
        </h2>
        <p />
        <hr />
      </div>
    </Link>
    <Link
      key="ergerg"
      replace={false}
      to={
        Object {
          "pathname": "/prepxp/blog/What-Google-and-employers-are-looking-for",
          "state": Object {
            "post": Object {
              "byline": "yesyesyeys",
              "id": "ergerg",
              "img": "http://via.placeholder.com/300x300",
              "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
              "title": "What Google and employers are looking for",
            },
            "posts": Array [
              Object {
                "byline": "yesyesyeys",
                "id": "1234",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf yea oia the ruler is bakc ofi course is i sfsf aoosd sgodf sfoeg wefodfguonfls sdfsd fsd sgosfsd sdfpsdf sdgxfogfdng  sdfosf sgosf sdfosg sdosf sgosdf sdfosdg sfosdf sd",
                "title": "5 Tips to get things done",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "134t",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Pass the interview with these answers",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "fddh",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "Top 10 most common questions",
              },
              Object {
                "byline": "yesyesyeys",
                "id": "ergerg",
                "img": "http://via.placeholder.com/300x300",
                "text": "sf sdfg dfasd df dg asdf sdf adsf sadf",
                "title": "What Google and employers are looking for",
              },
            ],
          },
        }
      }
    >
      <div
        className="blogPost__sidebar__item"
      >
        <h2>
          What Google and employers are looking for
        </h2>
        <p />
        <hr />
      </div>
    </Link>
  </div>
</div>
`;
